components/HomeNavbar.js

```diff
@@ -0,0 +1,46 @@
+import { Component } from 'react'
+
+class HomeNavbar extends Component {
+
+  constructor (props) {
+    super(props)
+    this.burgerClick = this.burgerClick.bind(this)
+  }
+
+  burgerClick () {
+    const nav = document.querySelector("ul[data-behavior='nav_links']")
```

Comment:
You'll want to use state for this, shouldn't be querying the DOM, we can go through this together